AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -261,10 +274,6 @@ - (void)setAsyncDataSource:(id<ASCollectionViewDataSource>)asyncDataSource
     _asyncDataSourceImplementsConstrainedSizeForNode = NO;
   } else {
     _asyncDataSource = asyncDataSource;
-    // TODO: Support supplementary views with ASCollectionView.
-    if ([_asyncDataSource respondsToSelector:@selector(collectionView:viewForSupplementaryElementOfKind:atIndexPath:)]) {
-      ASDisplayNodeAssert(NO, @"ASCollectionView is planned to support supplementary views by September 2015.  You can work around this issue by using standard items.");
```

Comment:
I knew it was risky to call out September - but this is a high-value feature man, people are going to be stoked to use this!